### DISCUSSION TITLE
Connecting Altherma EHVX08S23EJ6V

### DISCUSSION THREAD

**Discussion by marcomow** [revk/xESP32-Faikout#568] [Q&A]
Hello, I've researched far and wide how to connect an Altherma EHVX08S23EJ6V, without finding an answer. 

It has a BRP069A78 wifi gateway.

Here is the photo of the board, any idea what connector might be needed?
![IMG_20241202_134703](https://github.com/user-attachments/assets/64058c04-fb1b-4602-9807-33ac25f3e55c)
![IMG_20241202_134834](https://github.com/user-attachments/assets/68967bdf-0b60-4f53-a3b5-a455b3ececab)
![IMG_20241202_135533](https://github.com/user-attachments/assets/a4586925-7fda-4fbe-be15-1de02510fdf8)
![IMG_20241202_135523](https://github.com/user-attachments/assets/fa1d1a5d-65bf-4ad0-afed-0dccfbe7d68b)
![IMG_20241202_135517](https://github.com/user-attachments/assets/4185bc37-5d98-414c-b42d-824cd33aac28)
![IMG_20241202_135509](https://github.com/user-attachments/assets/f0c4061a-a696-46cb-9a54-52d93918180b)

**Comment by marcomow** [comment]
If anyone has at least a clue that would be amazing as I'm currently in the UK so I wouldn't have to go through shipping complications, thanks!

**Comment by linhpd88** [comment]
You can try contact this guy: https://community.home-assistant.io/t/sell-air-conditioner-control-module-local-fast/961256/5
